### PR TITLE
doc: changed main documentation to markdown format

### PR DESCRIPTION
**Pull request by matthieusb**
Thanks for your work on this plugin.
I just wanted to add a bit of markdown to the README so that it is more readable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Installation
+
+First clone the repository repo in the custom zsh plugins folder:
+
+``` bash
+cd ~/.oh-my-zsh/custom/plugins
+git clone https://github.com/pkulev/zsh-rustup-completion.git rustup
+```
+
+Open ~/.zshrc, add plugin to plugins and check that autoloading enabled:
```
Comment (pkulev) [inline README.md]: This statement implies one codeblock with plugin declaration and autoloading after it. At separate block it seems like shell command to execute once after file editing.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+``` bash
+autoload -U compinit && compinit
+```
+
+Reload config:
+``` bash
+source ~/.zshrc
+```
+
+Enjoy it !
```
Comment (pkulev) [inline README.md]: Extra space.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Oh-my-zsh rustup completion plugin.
+
+Rustup version: 1.11.0 (e751ff9f8 2018-02-13)
+
+## Installation
+
+First clone the repository repo in the custom zsh plugins folder:
+
+``` bash
```
Comment (pkulev) [inline README.md]: Here and below, github markdown guide does not prohibit space between triple backticks and language, but ewerywhere I saw such constructions without space.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# zsh-rustup-completion
+
+Oh-my-zsh rustup completion plugin.
+
+Rustup version: 1.11.0 (e751ff9f8 2018-02-13)
```
Comment (pkulev) [inline README.md]: Let's emphasize version with bold, we can now!

**Review by pkulev — CHANGES_REQUESTED** [review]
Thanks, markdown is good idea!

Please, make fixes with fixup commit.

File: README.md (hunk 2 above)
Comment (matthieusb) [reply README.md]: Do you want me to remove the extra space below this statement ?

File: README.md (hunk 1 above)
Comment (matthieusb) [reply README.md]: I'll remove the sentence and make it clearer afterwards in the document.

File: README.md (hunk 3 above)
Comment (matthieusb) [reply README.md]: I'll remove the space just in case

File: README.md (hunk 4 above)
Comment (matthieusb) [reply README.md]: Done, I'll push

File: README.md (hunk 2 above)
Comment (pkulev) [reply README.md]: No, just remove space between `it` and `!`.

Comment (matthieusb) [reply README.md]: Oh sorry. Done !

**Review by pkulev — APPROVED** [review]

**Comment by pkulev** [comment]
Thank you!

**Comment by matthieusb** [comment]
Thanks for merging !

**Pull request merged**